source/_integrations/tplink.markdown

```diff
@@ -71,5 +71,6 @@ There is currently support for the following device types within Home Assistant:
 - KL125
 - KL130
 - KB130
+- KL400 (supports consumption sensors)
```

Comment:
```suggestion
- KL400
```

Some of the other ones do as well. Since we don't really document this we should probably leave it out.